spec/features/form_sections/teacher_training_data/edit_course_details_spec.rb

```diff
@@ -19,18 +19,17 @@
   context "trainee has existing course details" do
     background do
       given_a_trainee_exists_with_course_details
+      and_the_trainees_course_subject_specialism_exists
       given_i_am_on_the_review_draft_page
     end
 
-    context "when the feature flag is turned on", feature_use_subject_specialisms: true do
-      scenario "submitting with valid parameters" do
-        given_a_subject_specialism_is_available_for_selection
-        when_i_visit_the_course_details_page
-        and_i_enter_valid_subject_specialism_parameters
-        and_i_enter_valid_parameters
-        and_i_submit_the_form
-        and_the_course_details_are_updated
-      end
+    scenario "submitting with valid parameters" do
+      given_a_subject_specialism_is_available_for_selection
+      when_i_visit_the_course_details_page
+      and_i_enter_valid_subject_specialism_parameters
+      and_i_enter_valid_parameters
+      and_i_submit_the_form
+      and_the_course_details_are_updated
```

Comment:
this should read as `then_the_course_details_are_updated` `given, when/and then`. Might need to add an `alias_method` if it's used in some other scenarios